packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -532,6 +608,38 @@ test.serial('bulk operation BULK_GRANULE_REINGEST reingests list of granule IDs'
   });
 });
 
+test.serial('bulk operation BULK_GRANULE_REINGEST reingests list of granule IDs with a workflowName', async (t) => {
+  await setUpExistingDatabaseRecords(t);
+  const workflowName = t.context.workflowName;
+  const granuleModel = new Granule();
+  const granules = await Promise.all([
+    granuleModel.create(fakeGranuleFactoryV2({ granuleId: t.context.granuleIds[0] })),
+    granuleModel.create(fakeGranuleFactoryV2({ granuleId: t.context.granuleIds[1] })),
+  ]);
+
+  await bulkOperation.handler({
+    type: 'BULK_GRANULE_REINGEST',
+    envVars,
+    payload: {
+      ids: [
+        granules[0].granuleId,
+        granules[1].granuleId,
+      ],
+      workflowName,
+    },
+  });
+
+  t.is(reingestStub.callCount, 2);
+  reingestStub.args.forEach((callArgs) => {
+    const matchingGranule = granules.find((granule) => granule.granuleId === callArgs[0].granuleId);
+    t.true(t.context.executionArns.includes(callArgs[0].execution));
+    delete matchingGranule.execution;
+    delete callArgs[0].execution;
```

Comment:
Because the model granule that I created in set up has a different executionArn than the function was called with, which would cause deepEqual to fail.   L635 checks that I called reingest with an execution that was in the RDB.  Can I do something to clear this up?

Reply:
Ah ok, that makes sense. Maybe just a one-line comment explaining why? I don't feel super strongly about it.

Reply:
sure 69591d2b7e